libs/platform/experience/src/services/layout/plugins/types/navigation/navigation-layout.plugin.ts

```diff
@@ -0,0 +1,30 @@
+import { ssrAwaiter } from '@spryker-oryx/core/utilities';
+import { Observable, of } from 'rxjs';
+import { LayoutStyles } from '../../../layout.model';
+import {
+  LayoutPlugin,
+  LayoutPluginConfig,
+  LayoutPluginOptionsParams,
+} from '../../layout.plugin';
+
+export class NavigationLayoutPlugin implements LayoutPlugin {
+  getStyles(data: LayoutPluginOptionsParams): Observable<LayoutStyles> {
+    const { options } = data;
+
+    return ssrAwaiter(
```

Comment:
Is it a right usage of `ssrAwaiter`? I mean, why should layout know something about ssr?

Reply:
yup, this is standard approach we took in the plugins to ensure we can lazy load styles.